webapp/src/containers/createRequest/index.tsx

```diff
@@ -0,0 +1,197 @@
+        setSuggestionChosen(true);
+        setRequest(requestSuggestion);
+    };
+
+    // Get config state
+    const getConfigState = () => {
+        const {isLoading, isSuccess, isError, data, error: apiErr} = getApiState(Constants.pluginApiServiceConfigs.getConfig.apiServiceName);
+        return {isLoading, isSuccess, isError, data: data as ConfigData | undefined, error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
+    };
+
+    const serviceNowBaseURL = getConfigState().data?.ServiceNowBaseURL;
+
+    return (
+        <Modal
+            show={isCreateRequestModalOpen(pluginState)}
+            onHide={hideModal}
+            className='servicenow-rhs-modal'
+        >
+            <>
+                <ModalHeader
+                    title='Begin Catalog Request'
+                    onHide={hideModal}
+                    showCloseIconInHeader={true}
+                />
+                <div
+                    className={`padding-h-12 padding-top-25 
+                    ${suggestionChosen ? 'height-290' : 'height-120'}`}
+                >
+                    <AutoSuggest
+                        placeholder='Search catalog items'
+                        inputValue={autoSuggestValue}
+                        onInputValueChange={setAutoSuggestValue}
+                        onChangeSelectedSuggestion={handleRequestSelection}
+                        disabled={showModalLoader}
+                        suggestionConfig={{
+                            suggestions,
+                            renderValue: (suggestion) => suggestion.name,
+                        }}
+                        charThresholdToShowSuggestions={Constants.CharThresholdToSuggestRequest}
+                    />
+                    {suggestionChosen && request && (
+                        <>
+                            <ul className='search-panel__description margin-top-25 padding-left-15 font-14'>
+                                {Constants.RequestDataLabelConfig.map((header) => (
+                                    <li
+                                        key={header.key}
+                                        className='d-flex align-items-center search-panel__description-item margin-bottom-10'
+                                    >
+                                        <span className='search-panel__description-header margin-right-10 text-ellipsis'>{header.label}</span>
+                                        <span className='search-panel__description-text channel-text wt-500 text-ellipsis'>{showModalLoader ? <SkeletonLoader/> : request[header.key] || 'N/A'}</span>
+                                    </li>
+                                ))}
+                            </ul>
+                            {serviceNowBaseURL && (
+                                <div>
+                                    <a
+                                        className='color--link btn btn-primary servicenow-request-button'
```

Comment:
```suggestion
                                        className='color-link btn btn-primary servicenow-request-button'
```
util classes can follow kebab-case

Reply:
i think this is an already defined mattermost class